db/active_replicator.go

```diff
@@ -34,11 +34,16 @@ type ActiveReplicator struct {
 // NewActiveReplicator returns a bidirectional active replicator for the given config.
 func NewActiveReplicator(ctx context.Context, config *ActiveReplicatorConfig) *ActiveReplicator {
 	ar := &ActiveReplicator{
-		ID:        config.ID,
-		config:    config,
-		statusKey: replicationStatusKey(config.ID),
+		ID:     config.ID,
+		config: config,
 	}
 
+	metakeys := base.DefaultMetadataKeys
+	if config.ActiveDB != nil {
```

Comment:
I was just being defensive in case tests were doing anything non-standard, but it's fair to say that the rest of the replicator code treats ActiveDB as always being non-nil.  I could drop the check.